src/util/loanSearch/filters/distributionModels.js

```diff
@@ -63,7 +66,7 @@ export default {
 	},
 	getRemovedFacet: () => ({ distributionModel: null }),
 	getSavedSearch: loanSearchState => ({
-		distributionModel: distributionModelEnumMap[loanSearchState?.distributionModel?.toUpperCase()]
+		distributionModel: distributionModelEnumMap[loanSearchState?.distributionModel?.toUpperCase() ?? 'BOTH']
```

Comment:
Is this long needed with the value map change?

Reply:
hmm prob not... This may have been my first change as I wanted to make sure a default was always applied.